server/controllers/changeSignedOperationalCapacity/index.ts

```diff
@@ -0,0 +1,123 @@
+import { NextFunction, Response } from 'express'
+import FormWizard from 'hmpo-form-wizard'
+import backUrl from '../../utils/backUrl'
+import FormInitialStep from '../base/formInitialStep'
+
+export default class ChangeSignedOperationalCapacity extends FormInitialStep {
+  middlewareSetup() {
+    super.middlewareSetup()
+    this.use(this.getSignedOperationalCapacity)
+  }
+
+  async getSignedOperationalCapacity(req: FormWizard.Request, res: Response, next: NextFunction) {
+    const { user } = res.locals
+    const { locationsService, manageUsersService } = req.services
+
+    const token = await req.services.authService.getSystemClientToken(user.username)
+    res.locals.signedOperationalCapacitySummary = await locationsService.getSignedOperationalCapacity(
+      token,
+      res.locals.prisonId,
+    )
+    res.locals.currentSignedOperationalCapacity = res.locals.signedOperationalCapacitySummary?.signedOperationCapacity
+    res.locals.whenUpdated = new Date(res.locals.signedOperationalCapacitySummary?.whenUpdated)
+    res.locals.whenUpdatedWeekday = res.locals.whenUpdated.toLocaleString('en', { weekday: 'long' })
+    res.locals.whenUpdatedDay = res.locals.whenUpdated.getDate()
+    res.locals.whenUpdatedMonth = res.locals.whenUpdated.toLocaleString('en', { month: 'long' })
+    res.locals.whenUpdatedYear = res.locals.whenUpdated.toLocaleString('en', { year: 'numeric' })
+    res.locals.whenUpdatedTime = res.locals.whenUpdated.toLocaleString('en', { timeStyle: 'short', hour12: false })
+
+    res.locals.residentialSummary = await locationsService.getResidentialSummary(token, res.locals.prisonId)
+    res.locals.maxCapacity = res.locals.residentialSummary?.prisonSummary?.maxCapacity
+
+    const { updatedBy } = res.locals.signedOperationalCapacitySummary || {}
+    res.locals.updatedBy = updatedBy
+      ? (await manageUsersService.getUser(res.locals.user.token, updatedBy))?.name || updatedBy
+      : updatedBy
```

Comment:
Can any of these just be variables instead of locals?

Reply:
Yes sorry, kinda just put stuff into locals whilst putting it together and then didn't go back to consider any changes. I have removed `signedOperationalCapacitySummary`, `whenUpdated`, and `residentialSummary` form locals.